documentation/bolt_installing.md

```diff
@@ -216,6 +216,11 @@ The Puppet Tools repository for the YUM package management system is [http://yum
         sudo rpm -Uvh https://yum.puppet.com/puppet-tools-release-fedora-30.noarch.rpm
         sudo dnf install puppet-bolt
         ```
+    -   Fedora 31
+        ```shell script
+        sudo rpm -Uvh https://yum.puppet.com/puppet-tools-release-fedora-31.noarch.rpm
```

Comment:
This release package doesn't seem to exist.

Reply:
The open PRs for `bolt-vanagon` and `ci-job-configs` need to be merged in and then once we start the release pipeline for Bolt the package will exist.

Reply:
Will it? I think that's what installs the Puppet repository for users to download from, not Bolt itself. It looks like PDK already ships a Fedora 31 package but you can't install it because you can't install the repo yet.

Reply:
I forgot - we'll need to ship the puppet-tools-release using the vanagon-generic-builder pipeline. That can be run as soon as Bolt packages are shipped.

Reply:
Repo for fedora 31 is available now: http://yum.puppet.com/